services/repository/src/main/java/org/sagebionetworks/auth/controller/AuthenticationController.java

```diff
@@ -123,21 +123,32 @@ Session authenticate(
 			@RequestBody LoginCredentials credentials)
 			throws NotFoundException {
 		LoginRequest request = DeprecatedUtils.createLoginRequest(credentials);
-		LoginResponse loginResponse =  authenticationService.login(request);
+		LoginResponse loginResponse =  authenticationService.loginForSession(request);
 		return DeprecatedUtils.createSession(loginResponse);
 	}
 
+	@Deprecated
+	@RequiredScope({})
+	@ResponseStatus(HttpStatus.CREATED)
+	@RequestMapping(value = UrlHelpers.AUTH_LOGIN, method = RequestMethod.POST)
+	public @ResponseBody
+	LoginResponse loginForSessionToken(@RequestBody LoginRequest request) throws NotFoundException {
+		return authenticationService.loginForSession(request);
+	}
+	
 	/**
-	 * Retrieve a session token that will be usable for 24 hours or until
-	 * invalidated. The user must accept the terms of use before a session token
-	 * is issued.
+	 * Retrieve an access token that will be usable for 24 hours. 
+	 * The user must accept the terms of use before the access token
+	 * can be used.
 	 */
 	@RequiredScope({})
 	@ResponseStatus(HttpStatus.CREATED)
-	@RequestMapping(value = UrlHelpers.AUTH_LOGIN, method = RequestMethod.POST)
+	@RequestMapping(value = UrlHelpers.AUTH_LOGIN_2, method = RequestMethod.POST)

```

Comment:
Can we double check if a "standard" endpoint name convention exists? E.g. what other services use or maybe specs?

Reply:
The [OpenID Connect discovery document spec'](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata) has no definition of the authentication endpoint.  I think we can use what we wish.  I am following the convention of our [`wiki2`](https://rest-docs.synapse.org/rest/#org.sagebionetworks.repo.web.controller.V2WikiController) and [`entitybundle2`](https://rest-docs.synapse.org/rest/#org.sagebionetworks.repo.web.controller.EntityBundleV2Controller) services, i.e. putting a '2' at the end of the URI.